src/mdo/java/WrapperList.java

```diff
@@ -27,16 +27,21 @@
 import java.util.function.Supplier;
 
 class WrapperList<T, U> extends AbstractList<T> {
```

Comment:
What's the point is the `WrapperList` itself does not implement `Serializable` ?